pkg/alerter/alerter.go

```diff
@@ -0,0 +1,79 @@
+package alerter
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/sirupsen/logrus"
+)
+
+type Alerter struct {
+	sync.Mutex
+	Alerts   Alerts
+	AlertsCh chan Alerts
+}
+
+func NewAlerter() *Alerter {
+	return &Alerter{
+		Alerts:   make(Alerts),
+		AlertsCh: make(chan Alerts),
+	}
+}
+
+func (a *Alerter) Start() {
```

Comment:
Do we really need a `Start` method ? Could we move this in the `NewAlerter` ?

Reply:
:+1:

Reply:
Let's remove it. I assumed it would have confused you to have both the initialization and the goroutine inside it. I was wrong :)

Reply:
Removed 7f5211c